pkgs/development/lua-modules/aliases.nix

```diff
@@ -43,4 +43,5 @@ mapAliases {
   cyrussasl = throw "cyrussasl was removed because broken and unmaintained "; # added 2023-10-18
   nlua-nvim = throw "nlua-nvim was removed, use neodev-nvim instead"; # added 2023-12-16
   nvim-client = throw "nvim-client was removed because it is now part of neovim"; # added 2023-12-17
+  toml = throw "toml was removed because unused. Feel free to restore"; # added 2024-06-25
```

Comment:
suggestion: add a note that toml-edit can also be used

Reply:
done